bool mode
int64 taskId
string taskName
---
bool ret
